utils/tests/test_cli.go

```diff
@@ -52,21 +52,32 @@ func (cli *JfrogCli) RunCliCmdWithOutput(t *testing.T, args ...string) string {
 	return RunCmdWithOutput(t, func() error { return cli.Exec(args...) })
 }
 
-// Run a command, redirect the stdout and return the output
-func RunCmdWithOutput(t *testing.T, executeCmd func() error) string {
+func (cli *JfrogCli) RunCliCmdWithOutputs(t *testing.T, args ...string) (string, error) {
+	return RunCmdWithOutputs(t, func() error { return cli.Exec(args...) })
+}
+
+func RunCmdWithOutputs(t *testing.T, executeCmd func() error) (output string, err error) {
 	newStdout, stdWriter, cleanUp := redirectOutToPipe(t)
 	defer cleanUp()
 
 	go func() {
-		assert.NoError(t, executeCmd())
+		err = executeCmd()
```

Comment:
it a gorutine. so this error won't return

Reply:
its a named output variable, also we are using the output and reading all the content right after running it.
maybe `io.ReadAll` is blocking? I did not change the logic just extracted it to a method